widgets/src/google/Common.js

```diff
@@ -90,8 +111,28 @@
                     italic: this._legendFontItalic
                 }
             },
+            series: initSeries(this.getNumSeries()),
+            axes: {
```

Comment:
google has axes & series just like the other charts same exact use for defining per axis params... here is an example of it being used in a scatter plot (which ill be adding soonish)

https://developers.google.com/chart/interactive/docs/gallery/scatterchart